src/components/ProcessCreate/Questions/Question.tsx

```diff
@@ -37,15 +37,15 @@ const Question = ({ index, remove }: Props) => {
         variant='transparent'
         position='absolute'
         right={2}
-        top={8}
+        top={7}
         type='button'
         icon={<DeleteIcon />}
         aria-label={t('form.process_create.question.question_delete', { values: { num: index } })}
         onClick={() => remove(index)}
         ml='auto'
       />
 
-      <FormControl isInvalid={isInvalidFieldMap(errors, `questions.${index}.title`)} mb={2}>
+      <FormControl isInvalid={isInvalidFieldMap(errors, `questions.${index}.title`)} mb={2} maxW='calc(100% - 50px)'>
```

Comment:
Using `calc` here isn't a good idea. You have full control over the layout and should be able to fix it using a "normal flexbox layout", no need for calc at all.

Also, try to add a condition to the remove button to only render when questions > 1